### DISCUSSION TITLE
How to reset filter webpart when using verticals?

### DISCUSSION THREAD

**Discussion by myatix** [microsoft-search/pnp-modern-search#1502] [Q&A]
Hi All,

I have set up a verticals webpart with a number of tabs and results webpart for each related tab on the verticals webpart.

Each results webpart has a standard query that queries items in a specific document library as well as their own "refinement filter" that sets the results shown based on the vertical you select. 

In addition to this I also have a refiner webpart that is used to reduce the results shown when a user selects a specific verticals tab.

My Issue is that as you click through the verticals tabs showing the various results webparts with their individual "refinement filters" the refiner webpart doesn't update??? It shows the same refiners as set in the first vertical???

![image](https://user-images.githubusercontent.com/48433526/140950911-b0ec4be2-1929-4727-a27a-bc4f3720356b.png)

![image](https://user-images.githubusercontent.com/48433526/140951328-5de15c42-7efc-468b-90ce-6a6562af9d9b.png)

![image](https://user-images.githubusercontent.com/48433526/140951459-672b1540-4c5e-477c-aed2-0d2adaba746b.png)

In other words as the results changed based on the vertical the user selects the refiner webpart doesn't update to reflect the new results?

Any ideas?
